core/src/test/java/feast/core/it/DataGenerator.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2018-2020 The Feast Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package feast.core.it;
+
+import com.google.common.collect.ImmutableList;
+import feast.proto.core.SourceProto;
+import feast.proto.core.StoreProto;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.apache.commons.lang3.tuple.Triple;
+
+public class DataGenerator {
```

Comment:
Is this supposed to be a generic data generator, because it seems very specific to stores/sources.

Reply:
Supposed to generic. Moved fetureSet creation here as well